web/cobrands/isleofwight/assets.js

```diff
@@ -0,0 +1,347 @@
+(function(){
+
+if (!fixmystreet.maps) {
+    return;
+}
+
+fixmystreet.stopper_after = true;
+
+var defaults = {
+    http_options: {
+        url: "https://tilma.staging.mysociety.org/mapserver/iow",
+        params: {
+            SERVICE: "WFS",
+            VERSION: "1.1.0",
+            REQUEST: "GetFeature",
+            SRSNAME: "urn:ogc:def:crs:EPSG::27700"
+        }
+    },
+    format_class: OpenLayers.Format.GML.v3.MultiCurveFix,
+    max_resolution: {
+        'isleofwight': 0.5291677250021167,
+        'fixmystreet': 1.194328566789627
+    },
+    attributes: {
+        central_asset_id: 'central_asset_id',
+        site_code: 'site_code'
+    },
+    min_resolution: 0.00001,
+    asset_id_field: 'asset_id',
+    geometryName: 'msGeometry',
+    srsName: "EPSG:27700",
+    strategy_class: OpenLayers.Strategy.FixMyStreet,
+    body: "Isle of Wight Council"
+};
+
+var pin_prefix = fixmystreet.pin_prefix || document.getElementById('js-map-data').getAttribute('data-pin_prefix');
+
+var labeled_default = {
+    fillColor: "#FFFF00",
+    fillOpacity: 0.6,
+    strokeColor: "#000000",
+    strokeOpacity: 0.8,
+    strokeWidth: 2,
+    pointRadius: 6
+};
+
+var labeled_select = {
+    externalGraphic: pin_prefix + "pin-spot.png",
+    fillColor: "#55BB00",
+    graphicWidth: 48,
+    graphicHeight: 64,
+    graphicXOffset: -24,
+    graphicYOffset: -56,
+    backgroundGraphic: pin_prefix + "pin-shadow.png",
+    backgroundWidth: 60,
+    backgroundHeight: 30,
+    backgroundXOffset: -7,
+    backgroundYOffset: -22,
+    popupYOffset: -40,
+    graphicOpacity: 1.0,
+
+    label: "${asset_id}",
+    labelOutlineColor: "white",
+    labelOutlineWidth: 3,
+    labelYOffset: 65,
+    fontSize: '15px',
+    fontWeight: 'bold'
+};
+
+var labeled_stylemap = new OpenLayers.StyleMap({
+  'default': new OpenLayers.Style(labeled_default),
+  'select': new OpenLayers.Style(labeled_select)
+});
+
+fixmystreet.assets.add($.extend(true, {}, defaults, {
+    http_options: {
+        params: {
+            TYPENAME: "streets"
+        }
+    },
+    always_visible: true,
+    non_interactive: true,
+    asset_type: 'area',
+    road: true,
```

Comment:
I don't think you need road: true here, unless there's some only_send interaction I'm not aware of.